### PR TITLE
x86_64: add the latest QEMU and a disk image for the PCI test

### PR DESCRIPTION
**Pull request by liva**
This is for https://github.com/lkl/linux/pull/487

### CODE REVIEW

File: circleci/x86_64/Dockerfile [hunk 1]
```diff
@@ -2,5 +2,39 @@ FROM lkldocker/circleci:0.5
 LABEL authors="Hajime Tazaki <thehajime@gmail.com>, Octavian Purdila <tavi@cs.pub.ro>"
 
 RUN sudo apt-get update && \
-    sudo apt-get install -y libfuse-dev libarchive-dev linux-headers-4.4.0-97-generic valgrind libnuma-dev && \
+    sudo apt-get install -y libfuse-dev libarchive-dev linux-headers-4.4.0-97-generic valgrind libnuma-dev cloud-image-utils wget pkg-config libglib2.0-dev libpixman-1-dev linux-image-4.15.0-112-generic && \
```
Comment (tavip) [inline circleci/x86_64/Dockerfile]: Please add a newline to make this line more visible.

**Review by tavip — COMMENTED** [review]
It might be better to create a new docker image, e.g. qemu/x86_64. @thehajime what do you think?

**Comment by thehajime** [comment]
> It might be better to create a new docker image, e.g. qemu/x86_64. @thehajime what do you think?

though i was not thinking about that, I agree with that to have a new one.

**Comment by liva** [comment]
I moved the Dockerfile to qemu-x86_64/Dockerfile and added new lines.

**Review by tavip — APPROVED** [review]
LGTM

**Comment by thehajime** [comment]
I’m reworking the image build from the current manually building by local script to automated build using CircleCI (builds on docker hub is slow and uncontrollable).  Once this new image is ready I will let you know here.

**Comment by thehajime** [comment]
The image should be available at lkldocker/circleci-qemu-x86_64:v1.0

https://hub.docker.com/r/lkldocker/circleci-qemu-x86_64/tags

**Pull request merged**